samples/model_builder/create_and_import_dataset_text_sample.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_create_and_import_dataset_text_sample]
```

Comment:
I forgot what we decided on the region tag, but we might need to add a certain marker to make sure there will be no conflict with existing (or future) GAPIC samples.

Reply:
Proposal: use the region tag `aiplatform_sdk_*_sample` for MBSDK samples